rpm/archivematica/archivematica.spec

```diff
@@ -256,25 +256,19 @@ fi
 # MCP Server
 %post mcp-server
 mkdir -p /var/log/archivematica/MCPServer
-touch /var/log/archivematica/MCPServer/MCPServer.log
-touch /var/log/archivematica/MCPServer/MCPServer_debug.log
 chown -R archivematica:archivematica /var/log/archivematica/MCPServer
 systemctl daemon-reload
 
 # MCP Client
 %post mcp-client
 mkdir -p /var/log/archivematica/MCPClient
-touch /var/log/archivematica/MCPClient/MCPClient.log
-touch /var/log/archivematica/MCPClient/MCPClient_debug.log
 chown -R archivematica:archivematica /var/log/archivematica/MCPClient
 systemctl daemon-reload
 
 # Dashboard
 %post dashboard
 
 mkdir -p /var/log/archivematica/dashboard
-touch /var/log/archivematica/dashboard/dashboard.log
-touch /var/log/archivematica/dashboard/dashboard_debug.log
```

Comment:
Are you sure this stuff is not needed anymore? Why it would be there then?

Reply:
Those files are now created by archivematica-dashboard. Also, the file created by /etc/archivematica/dashboard.debug.log is dashboard.debug.log, not dashboard_debug.log